src/claimtrie.cpp

```diff
@@ -578,7 +677,10 @@ bool CClaimTrie::recursiveCheckConsistency(const CClaimTrieNode* node) const
     {
         if (recursiveCheckConsistency(it->second))
         {
-            vchToHash.push_back(it->first);
+            std::stringstream ss;
```

Comment:
This exists just to convert a char to a string?

Reply:
A normalized string, yes.

Reply:
To be clear, this fork implements normalization by removing the concept of a claimtrie of _chars_ and we instead have a claimtrie of _code points_, represented as chars (via utf8).